types/react-select/test/examples/Experimental.tsx

```diff
@@ -87,10 +95,10 @@ const daysContainerStyles = {
     paddingLeft: '2%',
 };
 
-const Group = (props: any) => {
-    const { Heading, getStyles, children, label, innerProps, headingProps, cx } = props;
+const Group = (props: GroupProps<DateOption, false>) => {
+    const { Heading, getStyles, children, label, headingProps, cx } = props;
     return (
-        <div aria-label={label} css={getStyles('group', props)} {...innerProps}>
+        <div aria-label={label as string} style={getStyles('group', props)}>
```

Comment:
Why was the typecast needed?

Reply:
`label` can be a `ReactNode`. This code isn't safe, but it's what's provided in the examples.

Reply:
ah jeeze 

okay, so something, someday to fix aria-label probably